src/status.c

```diff
@@ -738,6 +754,9 @@ status_request(struct view *view, enum request request, struct line *line)
 		return request;
 	}
 
+	if (show_untracked_only && view->parent && !main_status_exists(view->parent, LINE_STAT_UNTRACKED))
+		return REQ_VIEW_CLOSE;
```

Comment:
Even if the status can only ever have the main view as the parent it would still be good to have a `view->parent == &main_view` before calling `main_status_exists`.

Reply:
Wilco

Reply:
I like "Impossible Germany" :-)
https://www.youtube.com/watch?v=r_0BPqWpM_M